workflows/mag.nf

```diff
@@ -561,7 +561,8 @@ workflow MAG {
         * DAS Tool: binning refinement
         */
 
-        if ( params.refine_bins_dastool && !params.skip_metabat2 && !params.skip_maxbin2 ) {
+        // If any two of the binners are both skipped at once, do not run because DAS_Tool needs at least one
+        if ( params.refine_bins_dastool && !( ( params.skip_metabat2 && params.skip_maxbin2 ) || ( params.skip_metabat2 && params.skip_concoct ) || ( params.skip_maxbin2 && params.skip_concoct ) ) ) {
```

Comment:
```suggestion
        if ( params.refine_bins_dastool && [ params.skip_metabat2, params.skip_maxbin2, params.skip_concoct ].count(false) > 1 ) {
```
I believe this would be better, not an exponential increase of tests, but linear. I tested it with nextflow console, but with variables instead of params, but I expect it would work?

Reply:
Ooooh that's clever, I admittedly get stuck on how to do this! I will try to test locally before committing